decidim-forms/app/packs/src/decidim/forms/admin/forms.js

```diff
@@ -392,6 +393,8 @@ export default function createEditableForm() {
 
       autoLabelByPosition.run();
       autoButtonsByPosition.run();
+
+      initLanguageChangeSelect();
```

Comment:
With the mentioned refactoring effort, we would also need to change this to something like this to apply only to those fields that were added to this section (and not to re-add the event listener to all selects on the page):
```js
      initLanguageChangeSelect($field[0].querySelectorAll("select.language-change"));
```

Otherwise every time you add a new field, the event listeners are re-added to all existing select elements on the page which can cause some problems along the way and also adds extra load to the front-end.

Reply:
Done.